pos_payment_entries_globalization/models/pos_session.py

```diff
@@ -91,6 +92,7 @@ def _generate_globalization_entries(self):
             if counterpart_credit:
                 self._create_globalization_counterpart_line(
                     0.0, counterpart_credit, global_account_id, move)
+        move._post_validate()
```

Comment:
Note this is done unconditionally when posting the move, so maybe it's better to not do it here. So in case of problem, the draft move can be fixed by the accountant when posting it?

Reply:
According to my tests it is possible after a difference between the debit and the credit to correct posted move lines